Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -1219,6 +1226,11 @@ void ApplicationManagerBase::CheckForIdle()
 
     if (CheckFullIdle())
     {
+        if (CheckReprocessFileList())
```

Comment:
so the theory here is
1/ the AP needs to get to Idle state
2/ then run the reprocess list

I assume that this list could be processed multiple times; 
or is it the ONLY list of assets that can be processed now?

Reply:
It could be processed multiple times, but the idea is it's a list you want to specifically process in this order, to debug issues that occur in that situation.